### PR TITLE
koord-scheduler: add e2e case for multi quota tree

### PR DESCRIPTION
**Pull request by shaloulcy**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: test/e2e/quota/multi_tree.go [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+				runtime, err := extension.GetRuntime(quota)
+				framework.ExpectNoError(err)
+				if quotav1.Equals(runtime, quota.Spec.Min) {
+					framework.Logf("child 1 runtime quota is same to min, runtime: %v", MarshalValue(runtime))
+					return true
+				}
+				return false
+			}, 60*time.Second, 1*time.Second).Should(gomega.Equal(true))
+
+			ginkgo.By("create quota profile2 with resource ratio")
```
Comment (eahydra) [inline test/e2e/quota/multi_tree.go:165]: There seem to be two scenarios, but they are both written together. It is recommended to split them into different cases.

File: test/e2e/quota/multi_tree.go [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+			err = f.SchedClientSet.SchedulingV1alpha1().ElasticQuotas(f.Namespace.Name).Delete(context.TODO(), childQuota1.Name, metav1.DeleteOptions{})
+			framework.ExpectNoError(err)
+
+			err = f.SchedClientSet.SchedulingV1alpha1().ElasticQuotas(f.Namespace.Name).Delete(context.TODO(), childQuota2.Name, metav1.DeleteOptions{})
+			framework.ExpectNoError(err)
+		})
+	})
+})
+
+func MarshalValue(val interface{}) string {
```
[the remark this answers is not in the document]
Comment (shaloulcy) [reply test/e2e/quota/multi_tree.go]: done

File: test/e2e/quota/multi_tree.go (hunk 1 above)
Comment (shaloulcy) [reply test/e2e/quota/multi_tree.go]: done

**Pull request merged**